cosm-eng/changesets
# feat(CWC-4286): adding since option to add command

## What did you implement:
Implemented issue: [CWC-4286](https://experiencecosm.atlassian.net/browse/CWC-4286)

## How did you implement it:

Added the since option to the add command

## How can we verify it:

From a dev based branch we would see packages changes comparing to main

<img width="1308" alt="image" src="https://github.com/user-attachments/assets/bf666896-d753-4fbf-8367-15fb798753ba">

For a dev branch branch with the --since develop we would see only changes made in the feature branch

<img width="1502" alt="image" src="https://github.com/user-attachments/assets/5e7a241f-05f4-4419-9f06-94f8a9e0e355">

* Blurred some things, but can remove the images altogether. 

## Checklist
